flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/AccumulatingAlignedProcessingTimeWindowOperatorTest.java

```diff
@@ -1038,7 +1038,7 @@ private void assertInvalidParameter(long windowSize, long windowSlide) {
 
 	// ------------------------------------------------------------------------
 
-	private static class StatefulFunction extends RichProcessWindowFunction<Integer, Integer, Integer, TimeWindow> {
+	private static class StatefulFunction extends ProcessWindowFunction<Integer, Integer, Integer, TimeWindow> {
```

Comment:
With this we add two unused imports.

And yes, I don't like that we don't have checkstyle active on the tests.